tests/integration/utils.py

```diff
@@ -0,0 +1,27 @@
+import time
+
+from solana.rpc.api import Client
+from solana.rpc.types import RPCResponse
+
+DEFAULT_MAX_TIMEOUT = 30  # 30 seconds  pylint: disable=invalid-name
+
+
+def confirm_transaction(tx_sig: str, time_out: int = DEFAULT_MAX_TIMEOUT) -> RPCResponse:
+    """Confirm a transaction."""
+    client = Client("http://localhost:8899")
```

Comment:
Please create a fixture for the http client and include it in conftest.py and check if there is a connection as I have done here https://github.com/serum-community/pyserum/pull/25/files#diff-484462fced51d1a06b1d93b4a44dd535R23-R31.